.travis.yml

```diff
@@ -20,9 +20,9 @@ addons:
 before_install:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
 before_script:
-- bundle exec rake test:$TEST_SUITE:setup
+- bundle exec rake test:setup
```

Comment:
For consistency, should this be moved into a tools/ci/before_script.sh file?  Actually, I think in other repos we put them in `bin/ci` as opposed to `tools/ci`, so we can do that too

Reply:
Moved this to `bin/ci/before_script.sh` and moved the other `tools/ci/*` to `bin/ci/`

Reply:
I didn't realize this was the case for all of the provider repos (using `bin/ci`).  Shows how much I actually interact with them...

That said, seems like this isn't how the `manageiq-ui-classic` works either, so we might want to update that repo as well while we are at it.

Reply:
I also wonder if we are `source`'ing bash scripts if they don't belong in something like `lib/ci/` instead of `bin`

We are actually executing `bin/ci/after_script` where we are `source tools/ci/before_install.sh`

Reply:
And interesting thought, though, I wonder if we should then just have a top level `ci` folder instead.

Reason I say that, is I think of `lib` as something that would be included with the build, and `ci` really isn't something that should be include in prod (it probably is in practice though).

Reply:
Yeah good point @NickLaMuro, I'll just leave this in `bin/ci` and we can follow-up with this later

Reply:
Please consider removing `bin/*` from `.gitignore` if we actually have tools in `bin/` now.

Reply:
(I think the original reason for `tools/` is that rails puts random stuff under `bin/` which we don't really control.)